training/model_management/src/run_model_preprocess.py

```diff
@@ -82,19 +94,37 @@ def _validate_transformers_args(args):
         preprocess_args.update(download_details.get("tags", {}))
         preprocess_args.update(download_details.get("properties", {}))
     preprocess_args["task"] = task_name if task_name else preprocess_args.get("task")
-    preprocess_args["model_id"] = model_id if model_id else preprocess_args.get("model_id")
+    preprocess_args["model_id"] = (
```

Comment:
nit: if running black pass `--line-length 119 ` to prevent non-required formatting